src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -120,14 +122,7 @@ async Task<Contributor> GetAuthor()
                 else if (contributors.Count > 0)
                 {
                     // When author name is not specified, last contributor is author
-                    for (var i = contributionCommits.Count - 1; i >= 0; i--)
-                    {
-                        var user = await GetContributor(contributionCommits[i]);
-                        if (user != null && !excludes.Contains(user.Name))
-                        {
-                            return user;
-                        }
-                    }
+                    return contributors[contributors.Count - 1];
```

Comment:
@superyyrrzz to confirm the expected behavior, I remember he added this loop for a purpose

Reply:
After #4861, these 2 loops can be unified.